crates/nu-command/src/database/commands/into_sqlite.rs

```diff
@@ -270,50 +329,53 @@ fn nu_value_to_string(value: Value, separator: &str) -> String {
 // REAL. The value is a floating point value, stored as an 8-byte IEEE floating point number.
 // TEXT. The value is a text string, stored using the database encoding (UTF-8, UTF-16BE or UTF-16LE).
 // BLOB. The value is a blob of data, stored exactly as it was input.
-fn nu_type_to_sqlite_type(nu_type: Type) -> &'static str {
-    match nu_type {
-        Type::Int => "INTEGER",
-        Type::Float => "REAL",
-        Type::String => "TEXT",
-        Type::Bool => "TEXT",
-        Type::Nothing => "NULL",
-        Type::Filesize => "INTEGER",
-        Type::Date => "TEXT",
-        _ => "TEXT",
+fn nu_value_to_sqlite_type(val: &Value) -> Result<&'static str, ShellError> {
+    match val.get_type() {
+        Type::String => Ok("TEXT"),
+        Type::Int => Ok("INT"),
```

Comment:
`INT` may be a synonym but it's not one of the ones found in the sqlite docs here https://www.sqlite.org/datatype3.html

Reply:
https://www.sqlite.org/datatype3.html#determination_of_column_affinity

>If the declared type contains the string "INT" then it is assigned INTEGER affinity.

And even in [STRICT](https://www.sqlite.org/stricttables.html) tables,

>The datatype must be one of following:
>
>INT
>INTEGER
>REAL
>TEXT
>BLOB
>ANY

Reply:
Right, it looks like a synonym. My personal preference is to have INTEGER but it probably doesn't matter so I won't hold up this PR because of that.

Reply:
Ah ok, no problem, I've renamed it to `INTEGER`